bubblewrap.c

```diff
@@ -308,6 +309,7 @@ usage (int ecode, FILE *out)
   fprintf (out,
            "    --help                       Print this help\n"
            "    --version                    Print version\n"
+           "    --exec PATH                  Execute PATH instead of COMMAND\n"
```

Comment:
What exactly this means is not immediately obvious. If you do it this way round then I'd prefer it to be named more like the `--exec-filename` from #223, and I'd prefer its help text to be maybe something like this:

> --exec-filename PATH: Execute PATH instead of COMMAND, but with COMMAND in argv[0]

But I think this option might actually be easier to document clearly if you swapped it round and made it `--argv0`, like `ld.so --argv0` and bash `exec -a`:

> --argv0 NAME: Set argv[0] to NAME when executing COMMAND

so that for your use-case, you would be using `bwrap --argv0 date ... $(realpath $(which date)) [ARGS]` or something.

Or, for the arguably more common use-case of making a shell think it's a login shell, `bwrap --argv0 -bash ... /bin/bash [ARGS]`.

Reply:
Thank you for the prompt and detailed review. I agree that the argv0 case is both easier to think about, and is more likely.

To document our decision, I’ll make the trivial changes to this pull request renaming —exec to —exec-filename, fixing the repeat failure, and making a short commit message, and will then close the pull request.

I will then also create a new pull request adding the —argv0 option.

Thank you for your advice!